service/application/src/main/resources/application.yml

```diff
@@ -1,3 +1,10 @@
+springdoc:
+  packages-to-scan: com.solace.maas.ep.runtime.agent.scanManager
```

Comment:
Wouldn't this be "com.solace.maas.ep.runtime.agent.scanManager.rest"?

Reply:
Thanks! `packages-to-scan` has been updated.